packages/web3-eth-personal/test/unit/eth_personal.test.ts

```diff
@@ -174,7 +174,10 @@ describe('EthPersonal', () => {
 			await ethPersonal.sendTransaction(tx, 'password');
 
 			expect(eth.formatTransaction).toHaveBeenCalledTimes(1);
-			expect(eth.formatTransaction).toHaveBeenCalledWith(tx, 'HexString');
+			expect(eth.formatTransaction).toHaveBeenCalledWith(tx, {
```

Comment:
Perhaps we can replace `{ bytes: 'BYTES_HEX', number: 'NUMBER_HEX' }` with `DEFAULT_RETURN_FORMAT`?

Reply:
Yes then we have to use `web3-common` as dependency for this package. I thought to avoid adding a dependency just for 2 test cases.